packages/cli/src/controller/init-controller.ts

```diff
@@ -138,12 +139,21 @@ export async function cloneProjectTemplate(
   return projectPath;
 }
 
-export async function readDefaults(projectPath: string): Promise<string[]> {
+export async function readDefaults(projectPath: string): Promise<{
+  endpoint: string | string[];
+  author: string;
+  description: string;
+  isMultiChainProject: boolean;
+}> {
   const packageData = await fs.promises.readFile(`${projectPath}/package.json`);
   const currentPackage = JSON.parse(packageData.toString());
-  let endpoint: ProjectNetworkConfig['endpoint'];
+  const author: string = currentPackage.author;
+  const description: string = currentPackage.description;
+  let endpoint: string[] | string;
```

Comment:
The endpoint can also be an object

https://github.com/subquery/subql/blob/main/packages/types-core/src/project/types.ts#L153